gtfs/spec/en/reference.md

```diff
@@ -60,6 +60,7 @@ Presence conditions applicable to fields and files:
 * **Optional** - The field or file may be omitted from the dataset.
 * **Conditionally Required** - The field or file must be included under conditions outlined in the field or file description.
 * **Conditionally Forbidden** - The field or file must not be included under conditions outlined in the field or file description.
+* **Recommended** - The field or file may be omitted from the dataset, but it is a best practice to include it. The best practice should be considered before omission.
```

Comment:
Try to make the last sentence clearer - e.g https://datatracker.ietf.org/doc/html/rfc2119#section-3  "considered before omission" is vague.  

Second draft attempt: 

"Before omitting this field or file, the best practice should be carefully evaluated and the full implications should be understood."